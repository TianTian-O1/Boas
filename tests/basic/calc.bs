print(2 + 3 * 5)
print((10 - 5) / 2.5)