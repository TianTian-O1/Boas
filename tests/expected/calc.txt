17.00
2.00